File: cibcc/Go/step9-11/sect3/cibcc.go
package main

import (
	"os"
	"fmt"
)

func main() {
	if len(os.Args) != 2 {
		fmt.Println("引数の個数が正しくありません")
	}

	token = tokenize([]rune(os.Args[1]))
	node := program()

	codegen(node)
}

File: cibcc/c++/step9-11/sect3/cibcc.cpp
#include "cibcc.h"

int main(int argc, char* argv[]) {
	if (argc != 2) {
		std::cout << "引数の個数が正しくありません\n";
	return 1;
	}

	token = tokenize(argv[1]);
	Node *node = program();

	codegen(node);

	return 0;
}

